tests/cloudformation/checks/resource/aws/example_SecurityGroupRuleDescription/SecurityGroupRuleDescription-PASSED-2.yaml

```diff
@@ -1,17 +1,10 @@
-AWSTemplateFormatVersion: 2010-09-09
 Resources:
-  TargetSG:
```

Comment:
@twmartin Why remove the reference to this resource?

Reply:
It seemed like another opportunity to strip the unit test down to the bare bones. I can certainly add things back, if my sentiment on the value of minimal unit tests is not shared.